makefile

```diff
@@ -43,3 +43,10 @@ clean:
 	cd fpga && rm -rf build
 	cd kicker && rm -rf build
 	rm -rf generated-docs
+
+CLANG_FORMAT_BINARY=clang-format-10
+
+pretty-lines:
+	@git diff -U0 --no-color $(DIFFBASE) | python3 util/style/clang-format-diff.py -binary $(CLANG_FORMAT_BINARY) -i -p1
+	@git diff -U0 --no-color $(DIFFBASE) | black . --exclude "control/mtrain" 
+	# excludes mtrain to not touch a separate repo
```

Comment:
otherwise the ci complains about invalid git hashes between the submodules (+1 for monorepo I guess)

Reply:
pr is already up lol. Once #156 is merged I will delete this exclude line.